masonry/src/text/edit.rs

```diff
@@ -188,23 +188,18 @@ impl<T: EditableText> TextEditor<T> {
                         }
                         Key::Named(_) => Handled::No,
                         Key::Character(c) => {
-                            let selection = self.inner.selection.unwrap_or(Selection {
-                                anchor: 0,
-                                active: 0,
-                                active_affinity: Affinity::Downstream,
-                                h_pos: None,
-                            });
-                            self.text_mut().edit(selection.range(), &**c);
-                            self.inner.selection = Some(Selection::caret(
-                                selection.min() + c.len(),
-                                // We have just added this character, so we are "affined" with it
-                                Affinity::Downstream,
-                            ));
-                            let contents = self.text().as_str().to_string();
-                            ctx.submit_action(Action::TextChanged(contents));
-                            Handled::Yes
+                            self.insert_text(event.text.as_ref().unwrap_or(c), ctx)
+                        }
+                        Key::Unidentified(_) => {
+                            if cfg!(windows) {
```

Comment:
Why is this `cfg!(windows)`? This at least needs an explanatory comment.

Reply:
I actually don't know that. I almost exclusively use Windows and do not own a Mac. So I don't know what the behavior should be for Linux or macOS.

Reply:
Thanks!

Reply:
Thanks!